net-misc/openssh/openssh-8.5_p1-r3.ebuild

```diff
@@ -132,6 +132,7 @@ src_prepare() {
 	eapply "${FILESDIR}"/${PN}-7.5_p1-disable-conch-interop-tests.patch
 	eapply "${FILESDIR}"/${PN}-8.0_p1-fix-putty-tests.patch
 	eapply "${FILESDIR}"/${PN}-8.0_p1-deny-shmget-shmat-shmdt-in-preauth-privsep-child.patch
+	eapply "${FILESDIR}"/${PN}-8.7p1-upstream-cve-2021-41617.patch
```

Comment:
There’s a typo in the actual patch file name so this won’t apply

Reply:
Just fixed the patch name. Thank you!